includes/init.php

```diff
@@ -32,8 +32,10 @@
 chdir($install_dir);
 
 // Libraries
-require('Net/IPv4.php');
-require('Net/IPv6.php');
+if (!module_selected('phpunit', $init_modules)) {
```

Comment:
How did you do it before? We ask users to install them via pear or apt-get normally.